jsapp/js/app.es6

```diff
@@ -21,8 +21,8 @@ import {
 import moment from 'moment';
 import {actions} from './actions';
 import {stores} from './stores';
-// importing it so it exists
-import {surveyCompanionStore} from './surveyCompanionStore';
+import {surveyCompanionStore} from './surveyCompanionStore'; // importing it so it exists
+import {alertify} from './alertify'; // importing it so it exists
```

Comment:
Oh right! Initially I thought that I will need this file to just define new confirm for alertify and never use it again. But as it turned out I do import this elsewhere so it does exist. Forgot to remove this line :)

As for `surveyCompoanionStore` - this file is not imported in any other place. If it is not imported anywhere then it would not be initialized and the store will simply not work.